cekit/generator/base.py

```diff
@@ -60,11 +62,18 @@ def __init__(self, descriptor_path, target, overrides):
 
                 LOGGER.debug("Loading override '{}'".format(override))
 
-                override_artifact_dir = os.path.dirname(os.path.abspath(override))
-                if not os.path.exists(override):
-                    override_artifact_dir = os.path.dirname(os.path.abspath(descriptor_path))
-                self._overrides.append(Overrides(tools.load_descriptor(
-                    override), override_artifact_dir))
+                if urlparse(override).scheme:
```

Comment:
Isn't here missing something? Like the scheme comparison?

Reply:
I was relying on the fact, if the scheme is filled in by `urlparse` and not an empty string, then download_file can handle the rest ; if its an unsupported scheme then download_file will detect that and throw an exception.

**However**, saying that it looks like `urlparse` gets really confused when it sees e.g. `labels:` and assumes that is a URL scheme (it mustn't check for `//`) .